demos/twa-play-billing/build.gradle

```diff
@@ -0,0 +1,49 @@
+/*
+ *    Copyright 2019 Google LLC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "29.0.2"
+    defaultConfig {
+        applicationId "com.google.androidbrowserhelper.demos.playbilling"
+        minSdkVersion 23
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
```

Comment:
A question: we sometimes remove proguard, others we leave it. I'm ok with both, but wondering if we have a reason to go one way or the other.

Reply:
I'll remove it - basically I'm happy to make demos have as few spurious lines as possible.

Reply:
It would also be good to make everything be consistent - I may do that once alpha is merged back into master, otherwise it could be a bit messy.